src/apps/reservation-list/list/reservation-list.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState, FC } from "react";
+import { useText } from "../../../core/utils/text";
+import { useGetReservationsV2 } from "../../../core/fbs/fbs";
+import { ReservationDetailsV2 } from "../../../core/fbs/model/reservationDetailsV2";
+
+const ReservationList: FC = () => {
+  const t = useText();
+  const { isSuccess, data } = useGetReservationsV2();
+  const [reservations, setReservations] = useState<ReservationDetailsV2[]>([]);
```

Comment:
You could also use null as initial value.
Then you could distinguish between initial value and value with the response.
If the response returns an empty array, you would know that `reservations ` is an empty array because the service said it was.
Does it make sense?

Reply:
yes, but I think I'll be using the [] for the ```<emptylist``` component